src/Logger/UpdateConsoleLogger.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the humbug/php-scoper package.
+ *
+ * Copyright (c) 2017 Théo FIDRY <theo.fidry@gmail.com>,
+ *                    Pádraic Brady <padraic.brady@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Humbug\PhpScoper\Logger;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @private
+ * @final
+ */
+class UpdateConsoleLogger
+{
+    /**
+     * @var SymfonyStyle
+     */
+    private $io;
+
+    public function __construct(SymfonyStyle $io)
+    {
+        $this->io = $io;
+    }
+
+    public function startUpdating()
+    {
+        $this->io->writeln('Updating...');
+    }
+
+    public function updateSuccess(string $newVersion, string $oldVersion)
+    {
+        $this->io->writeln('PHP-Scoper has been updated.');
```

Comment:
with `SymfonyStyle` you also have `success()`, `error()` and such for some fancy styling